src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
```

Comment:
should be removed, that's a false positive.

Reply:
Updated =)